applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js

```diff
@@ -149,17 +149,17 @@ return view.extend({
 		s.taboption('advanced', form.Value, 'serial_number', _('Announced serial number'))
 		s.taboption('advanced', form.Value, 'model_number', _('Announced model number'))
 
-		o = s.taboption('advanced', form.Value, 'notify_interval', _('Notify interval'))
+		o = s.taboption('advanced', form.Value, 'notify_interval', _('Notify interval'), _('A 900s interval gives notifies with max-age of 1800s, UDA min.'))
```

Comment:
`gives` is unclear here. Do you mean yields? (verb) yields 2nd person, and notifies plural noun reads confusingly the first few times. Please rethink this. Maybe
`To yield a notify of max-age 1800s, use a 900s interval`. 

What is UDA min? Expand and explain this abbreviation.